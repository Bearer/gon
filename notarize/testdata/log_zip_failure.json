{
  "logFormatVersion": 1,
  "jobId": "4ba7c420-7444-44bc-a190-1bd4bad97b13",
  "status": "Invalid",
  "statusSummary": "Archive contains critical validation errors",
  "statusCode": 4000,
  "archiveFilename": "gon.zip",
  "uploadDate": "2019-11-06T00:54:22Z",
  "sha256": "c109f26d378fbf1efadc8987fdab79d2ce63155e8941823d4d11a907152e11a5",
  "ticketContents": null,
  "issues": [
    {
      "severity": "error",
      "code": null,
      "path": "gon.zip/foo",
      "message": "The binary is not signed.",
      "docUrl": null,
      "architecture": "x86_64"
    },
    {
      "severity": "error",
      "code": null,
      "path": "gon.zip/foo",
      "message": "The signature does not include a secure timestamp.",
      "docUrl": null,
      "architecture": "x86_64"
    },
    {
      "severity": "error",
      "code": null,
      "path": "gon.zip/foo",
      "message": "The executable does not have the hardened runtime enabled.",
      "docUrl": null,
      "architecture": "x86_64"
    }
  ]
}
