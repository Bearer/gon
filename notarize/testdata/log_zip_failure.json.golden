(*notarize.Log)({
 JobId: (string) (len=36) "4ba7c420-7444-44bc-a190-1bd4bad97b13",
 Status: (string) (len=7) "Invalid",
 StatusSummary: (string) (len=43) "Archive contains critical validation errors",
 StatusCode: (int) 4000,
 ArchiveFilename: (string) (len=7) "gon.zip",
 UploadDate: (string) (len=20) "2019-11-06T00:54:22Z",
 SHA256: (string) (len=64) "c109f26d378fbf1efadc8987fdab79d2ce63155e8941823d4d11a907152e11a5",
 Issues: ([]notarize.LogIssue) (len=3 cap=4) {
  (notarize.LogIssue) {
   Severity: (string) (len=5) "error",
   Path: (string) (len=11) "gon.zip/foo",
   Message: (string) (len=25) "The binary is not signed."
  },
  (notarize.LogIssue) {
   Severity: (string) (len=5) "error",
   Path: (string) (len=11) "gon.zip/foo",
   Message: (string) (len=50) "The signature does not include a secure timestamp."
  },
  (notarize.LogIssue) {
   Severity: (string) (len=5) "error",
   Path: (string) (len=11) "gon.zip/foo",
   Message: (string) (len=58) "The executable does not have the hardened runtime enabled."
  }
 },
 TicketContents: ([]notarize.LogTicketContent) <nil>
})
